templates/jws50-tomcat9-basic-s2i.json

```diff
@@ -0,0 +1,323 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-rh-tomcat",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "tags": "tomcat,tomcat9,java,jboss",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss Web Server 5.0 Apache Tomcat 9 (no https)",
+            "description": "An example JBoss Web Server application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.0 Apache Tomcat 9 based application, including a build configuration, and an application deployment configuration.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-web-server/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "jws50-tomcat9-basic-s2i"
+    },
+    "labels": {
+        "template": "jws50-tomcat9-basic-s2i",
+        "jws": "1.0"
```

Comment:
maybe `jws50`, to distinguish from other versions?

Reply:
Changed the label to jws50.